api/wasm-interpreter/src/lib.rs

```diff
@@ -203,6 +207,39 @@ impl WrappedInstance {
         self.0.highlight(_path.into(), _offset);
         let _ = slint_interpreter::invoke_from_event_loop(|| {}); // wake event loop
     }
+
+    /// THIS FUNCTION IS NOT PART THE PUBLIC API!
+    /// Request information on what to highlight in the editor based on clicks in the UI
+    #[cfg(feature = "highlight")]
+    #[wasm_bindgen]
+    pub fn design_mode(&self, active: bool) {
```

Comment:
The name of this function reads like a getter. Did you mean to call it set_design_mode or enable_design_mode?